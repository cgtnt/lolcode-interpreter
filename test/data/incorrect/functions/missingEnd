HAI

HOW IZ I FOO
VISIBLE "hi"

KTHXBYE
