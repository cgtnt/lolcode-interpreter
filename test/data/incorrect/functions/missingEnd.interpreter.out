Line 6: Invalid expression, unexpected token KTHXBYE
Line 7: Expected tokens: IF U SAY SO
Line 7: Expected tokens: KTHXBYE
